### DISCUSSION TITLE
Update the guide for the the SBC installation

### DISCUSSION THREAD

**Discussion by DariusR10** [Berny23/LD-ToyPad-Emulator#122] [Ideas]
Update the guide bc i had to install many other programs(?) that are not in the guied and it's baerly working (it only worked onece, yesterday 15th of march and today it doesn't want to connect through SSH) after 1 week of installing and re-installing and running all the commands on the Pi Zero.

For example:
In step 1
Before running the `sudo apt install -y git` command i had to run the `sudo apt update` and then `sudo apt install -y git` or else it wouldn't work.

Or in the step 5,
before i run the `npm install` command, i had to run 2 other commands to install libsub-1.0-0-dev and lubudev-dev. After i installed thoes it WARNed me that i still had to install 2 peer depemdencies, how do i install thoes correctly?

Even tho the guide is easy to follow, as someone that has never programed anything there are things that are not explained in the guide, please update the guide and explain it with more detaill.

**Comment by Berny23** [comment]
I updated the guide in commit 8ff9881.
It's now working perfectly on my virtual machine. Peer dependencies are also installed correctly now.

**Comment by DariusR10** [comment]
Thank you for updating the guide!
